modules/ROOT/pages/nav-matrix.adoc

```diff
@@ -5,7 +5,7 @@ include::{partialsdir}/attributes.adoc[]
 [discrete]
 == Prerequisites
 
-* [x] You have xref:ROOT:index.adoc[set up AeroGear Mobile Services on OpenShift].
```

Comment:
not sure about this change @finp does this makes sense that we send the user back to the getting started guide?

Reply:
let's rip it out